### PR TITLE
Add documentation blurb about the istio operator

### PR DESCRIPTION
**Pull request by aallbrig**
This PR is to add a short blurb about the istio operator in the README, to help with the move to consolidate [istio/operator](https://github.com/istio/operator) into this repo.

### CODE REVIEW

**Comment by aallbrig** [comment]
@googlebot I signed it!

**Comment by aallbrig** [comment]
cc @linsun @sdake

File: README.md [hunk 1]
```diff
@@ -86,6 +86,11 @@ proxies, and collect telemetry data from proxies and services. There
 are plugins for interfacing with various cloud platforms, policy
 management services, and monitoring services.
 
+    - [operator](operator/). This directory contains code for
+[_istio operator_](https://istio.io/docs/reference/commands/operator/). You can read operator installation instructions
+[here](https://istio.io/docs/setup/install/standalone-operator/). If you have a bug backport for istio operator 1.4
```
Comment (sdake) [inline README.md]: The workflow stuff is a little suspect and will need to be removed once 1.4 is deprecated (which would be once this repo is released). This PR is backportable, however, so I'll submit a backport request, and then I recommend a followup PR to remove the "if you have a bug report.." part.

Comment (aallbrig) [reply README.md]: > a followup PR to remove the "if you have a bug report.." part

@sdake Sounds good to me. I'll work on this now.

**Comment by aallbrig** [comment]
@sdake I rebased this branch on the latest from istio.

It looks like the build system is waiting on an approval review, but this PR already has one -- yours. Is there something addition I need to do to sort this out?

**Comment by aallbrig** [comment]
I've rebased this branch with the latest from master from upstream istio/istio

@costinm @linsun @rshriram This PR should be ready for your consideration.

**Pull request closed without merging**